roles/www_base/files/html/html/credits.html

```diff
@@ -33,12 +33,18 @@ <h1>Internet-in-a-Box Credits</h1>
   All HealthPhone content is available for free at  <a href="http://www.healthphone.org/">www.healthphone.org</a>.<br>
   All Centers for Disease Control content is available for free at  <a href="https://www.cdc.gov/">www.cdc.gov</a>.<br>
   All Global Emergency Medicine Wiki content is available for free at  <a href="https://wikem.org/wiki/Main_Page">wikem.org/wiki/Main_Page</a>.<br><br>
+  All OpenStreetMap data is available for free at  <a href="https://www.openstreetmap.org/">www.openstreetmap.org</a>.<br><br>
+  All Natural Earth data is available for free at  <a href="https://www.naturalearthdata.com">www.naturalearthdata.com</a>.<br><br>
+  All Sentinel-2 satellite content is available for free (some under a non-commercial license) at <a href="https://s2maps.eu">s2maps.eu</a>.<br><br>
   
   Internet-in-a-Box also includes the work of content aggregators which we gratefully acknowledge:<br><br>
 
   Kiwix is a ZIM server and repository of Wikimedia and other content in a compressed ZIM file format at  <a href="https://www.kiwix.org/">www.kiwix.org</a>.<br>
   KA Lite is a server and repository of Khan Academy content in various languages at  <a href="https://learningequality.org/ka-lite/">learningequality.org/ka-lite</a>.<br><br>
   OER2Go/RACHEL is a curation of selected offline content at  <a href="https://rachel.worldpossible.org/content">rachel.worldpossible.org/content</a>.<br>
+  MapLibre is a set of mapping libraries available at <a href="https://maplibre.org">maplibre.org</>
```

Comment:
I wouldn't have mentioned this because it's covered by maps.black, but I use a MapLibre library for the search bar, globe icon etc.